doc/api/esm.md

```diff
@@ -388,49 +338,69 @@ can be written:
 }
 ```
 
-When using [Conditional Exports][], the rule is that all keys in the object
-mapping must not start with a `"."` otherwise they would be indistinguishable
-from exports subpaths.
+#### Conditional Exports
 
-<!-- eslint-skip -->
-```js
-{
-  "exports": {
-    ".": {
-      "import": "./main.js",
-      "require": "./main.cjs"
-    }
-  }
-}
-```
+Conditional exports provide a way to map to different paths depending on
+certain conditions. They are supported for both CommonJS and ES module imports.
 
-can be written:
+For example, a package that wants to provide different ES module exports for
+require() and import can be written:
 
 <!-- eslint-skip -->
 ```js
+// ./node_modules/pkg/package.json
 {
+  "main": "./main-require.cjs",
   "exports": {
-    "import": "./main.js",
-    "require": "./main.cjs"
-  }
+    "import": "./main-module.js",
+    "require": "./main-require.cjs"
+  },
+  "type": "module"
 }
 ```
 
-If writing any exports value that mixes up these two forms, an error will be
-thrown:
+The conditions supported in Node.js condition matching:
+
+* `"default"` - the generic fallback that will always match. Can be a CommonJS
+   or ES module file. _This condition should always come last._
```

Comment:
If `"default"` should always come last, then maybe it should be last in this list too?

Reply:
Makes sense, will do.